projects/libsodium/secretbox_easy_fuzzer.cc

```diff
@@ -0,0 +1,29 @@
+extern "C" {
+  #include <sodium.h>
+}
+
+class SodiumState {
+public:
+  unsigned char key[crypto_secretbox_KEYBYTES];
+  unsigned char nonce[crypto_secretbox_NONCEBYTES];
+
+  SodiumState() {
+    sodium_init(); // this can fail with a non-zero return code
```

Comment:
See the comment I made on secret_key_auth_fuzzer.cc:11. 
If `sodium_init` fails, it might be better to find out about it here rather than later in the code. 
But I'm unfamiliar with the situations in which this can happen, so it's up to you.

Reply:
Changing to include the assertion.